client/coral-embed-stream/src/Embed.js

```diff
@@ -144,7 +158,10 @@ class Embed extends Component {
       <div style={expandForLogin}>
         <div className="commentStream">
           <TabBar onChange={this.changeTab} activeTab={activeTab}>
-            <Tab><Count count={asset.totalCommentCount}/></Tab>
+            <Tab><Count
+              count={asset.totalCommentCount}
+              handleClick={this.handleClick}/>
```

Comment:
The issue is that this logic is also used by the Show all comments button. 

And I agree about the issue on isAdmin only refetching data. Let's work that out in other story.